lib/simplifyExpression/basicsSearch/removeMultiplicationByNegativeOne.js

```diff
@@ -1,56 +1,30 @@
-const clone = require('../../util/clone');
+const {rewriteNode} = require('math-rules');
 
 const ChangeTypes = require('../../ChangeTypes');
-const Negative = require('../../Negative');
+const defineRuleString = require('../../util/defineRuleString');
 const Node = require('../../node');
 
-// If `node` is a multiplication node with -1 as one of its operands,
-// and a non constant as the next operand, remove -1 from the operands
-// list and make the next term have a unary minus.
+// e.g. x * -1 -> -x
+const REMOVE_MULTIPLYING_BY_NEGATIVE_ONE = defineRuleString('#a * -1', '-#a');
+
+// e.g. -1 * x -> -x
+const REMOVE_MULTIPLYING_BY_NEGATIVE_ONE_REVERSE = defineRuleString('-1 * #a', '-#a');
+
+// If `node` is of the form 1^x, reduces it to a node of the form 1.
 // Returns a Node.Status object.
 function removeMultiplicationByNegativeOne(node) {
-  if (node.op !== '*') {
-    return Node.Status.noChange(node);
+  let newNode = null;
+  newNode = rewriteNode(REMOVE_MULTIPLYING_BY_NEGATIVE_ONE, node);
+  if (!newNode) {
+    newNode = rewriteNode(REMOVE_MULTIPLYING_BY_NEGATIVE_ONE_REVERSE, node);
   }
```

Comment:
This is not fun having to do these checks.  I'll see what I can do about this over the weekend.